ccm_web/server/src/health/health.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common'
+import { ConfigModule } from '@nestjs/config'
+import { TerminusModule } from '@nestjs/terminus'
+
+import { HealthController } from './health.controller'
+
+@Module({
+  controllers: [HealthController],
+  imports: [ConfigModule, TerminusModule]
```

Comment:
the Sequelize module needs to be imported in order to make this work. 
https://github.com/nestjs/terminus/blob/master/sample/007-sequelize-app/src/health/health.module.ts